### PR TITLE
Updating ML extension to use latest public preview for SDKv2

### PR DESCRIPTION
**Pull request by harneetvirk**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
ML extension

**Review by kairu-ms — APPROVED** [review]

File: src/index.json [hunk 1]
```diff
@@ -25079,6 +25079,73 @@
...
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions/tree/master/src/machinelearningservices"
```
Comment (jsntcy) [inline src/index.json:25114]: Could you update to the correct project url?

File: src/index.json [hunk 2]
```diff
@@ -25079,6 +25079,73 @@
...
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9",
+                        "Programming Language :: Python :: 3.10",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
```
Comment (jsntcy) [inline src/index.json:25105]: Could you please update to the correct contact email?

**Review by necusjz — APPROVED** [review]

File: src/index.json (hunk 2 above)
Comment (harneetvirk) [reply src/index.json:25105]: We were always having the same email in previous ml extension. I will update the email in next ML extension.

File: src/index.json (hunk 1 above)
Comment (harneetvirk) [reply src/index.json:25114]: We were always having the same url in previous ml extension. I will update the url in next ML extension.

Comment (harneetvirk) [reply src/index.json:25114]: The project URL has been updated.

**Pull request merged**